pkg/component/component.go

```diff
@@ -452,6 +452,15 @@ func PushLocal(client *occlient.Client, componentName string, applicationName st
 	}
 	targetPath := fmt.Sprintf("%s/src", s2iSrcPath)
 
+	stdin := bytes.NewBuffer([]byte{})
+	stdout := bytes.NewBuffer([]byte{})
+	stderr := bytes.NewBuffer([]byte{})
+	err = client.ExecCMDInContainer(pod.Name, strings.Split("/var/lib/supervisord/bin/supervisord ctl start run", " "), stdout, stderr, stdin, false)
```

Comment:
This seems super hacky, could we at least move the supervisord path to a constant?

Reply:
+1

Reply:
PR modified but principle maintained Please revisit the PR @cdrage @surajnarwade